blueprints/stylish-press/blueprint.json

```diff
@@ -6,14 +6,16 @@
 		"author": "adamziel",
 		"categories": ["Woocommerce", "Site"]
 	},
-	"landingPage": "/shop",
+	"landingPage": "/contact-us/",
```

Comment:
It's not a big deal, but it's a bit confusing to have contact-us here.

Could we add a _Shop_ page [to the wxr](https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/stylish-press/site-content.wxr) and use it as the shop page for Woo?

Reply:
It feels like something to discuss with the original developer of the blueprint. I could create an issue for this separately.

Reply:
I opened an issue for discussion https://github.com/WordPress/blueprints/issues/117